pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin.go

```diff
@@ -126,16 +124,6 @@ func applyToGateway(
 		return err
 	}
 	for _, listenerInfo := range gatewayListerInfos {
-		address := dataplane.Spec.GetNetworking().Address
```

Comment:
Do you know exactly why it didn't work? Was it caused by the wrong matching of listeners?

Reply:
These listeners are the exposed gateway listeners. We can't apply config to them based on a `to.targetRef`. The only way these listeners should be modified are if we have something like `spec.targetRef.kind: MeshGateway` and `from.targetRef.kind: NonMesh` or maybe `spec.targetRef.kind: MeshGatewayRoute`. This may be an issue with other policies as well.